### PR TITLE
Add button for selecting all files in the room

### PR DESCRIPTION
**Pull request by szero**
As above ^

### CODE REVIEW

File: admin.iptools.user.js [hunk 1]
```diff
@@ -115,8 +119,7 @@ body[noipspls] .tag_key_ip {
   const update = () => {
     if (state) {
       document.body.setAttribute("noipspls", "true");
-    }
-    else {
+    } else {
```
Comment (RealDolos) [inline admin.iptools.user.js]: >same line else
kys

File: admin.iptools.user.js [hunk 2]
```diff
@@ -70,7 +72,9 @@ dry.once("dom", () => {
   console.log(
     "running", GM.info.script.name, GM.info.script.version, dry.version);
 
-  const style = $e("style", {id: "iptools-style"}, `
+  const style = $e("style", {
```
Comment (RealDolos) [inline admin.iptools.user.js]: bls

File: admin.iptools.user.js [hunk 3]
```diff
@@ -20,7 +20,9 @@ dry.once("dom", () => {
   const IS_BAN = /^\S+ (?:banned|muted|hellbanned|unbanned|un-muted|timed-in)\b/;
 
   function adjustBanPart(p, users, ips) {
-    const {value} = p;
+    const {
```
Comment (RealDolos) [inline admin.iptools.user.js]: double bls

File: admin.iptools.user.js [hunk 4]
```diff
@@ -154,14 +157,12 @@ body[noipspls] .tag_key_ip {
           for (const el of elems) {
             el.textContent = res.name;
           }
-        }
-        catch (ex) {
+        } catch (ex) {
```
Comment (RealDolos) [inline admin.iptools.user.js]: >same line catch
KYS

File: admin.iptools.user.js [hunk 5]
```diff
@@ -154,14 +157,12 @@ body[noipspls] .tag_key_ip {
...
           console.error(ex);
         }
         roomqueue.delete(room);
       }
-    }
-    finally {
+    } finally {
```
Comment (RealDolos) [inline admin.iptools.user.js]: m(

File: admin.iptools.user.js [hunk 6]
```diff
@@ -175,17 +176,15 @@ body[noipspls] .tag_key_ip {
           if (message[0].type === "url" && message[0].href === "/reports") {
             if (message[1].value.startsWith(" / BLACKLIST ")) {
               adjustBanPart(message[4], users, ips);
-            }
-            else {
+            } else {
```
Comment (RealDolos) [inline admin.iptools.user.js]: m(

File: admin.iptools.user.js [hunk 7]
```diff
@@ -175,17 +176,15 @@ body[noipspls] .tag_key_ip {
...
               const [, p] = message;
               const m = p.value.match(/ \((\d+\.\d+\.\d+\.\d+)\)/);
               if (m) {
                 p.value = p.value.replace(m[0], "");
                 ips.push(m[1]);
               }
             }
-          }
-          else {
+          } else {
```
Comment (RealDolos) [inline admin.iptools.user.js]: 🇵🇱

File: admin.iptools.user.js [hunk 8]
```diff
@@ -208,8 +207,7 @@ body[noipspls] .tag_key_ip {
             data.ip = ips.join(" ip:");
           }
         }
-      }
-      catch (ex) {
+      } catch (ex) {
```
Comment (RealDolos) [inline admin.iptools.user.js]: 🇵🇱

File: admin.iptools.user.js [hunk 9]
```diff
@@ -244,18 +242,15 @@ body[noipspls] .tag_key_ip {
               const l = roomqueue.get(m[1]);
               if (!l) {
                 roomqueue.set(m[1], [el]);
-              }
-              else {
+              } else {
```
Comment (RealDolos) [inline admin.iptools.user.js]: 👳🏿

File: admin.iptools.user.js [hunk 10]
```diff
@@ -334,8 +337,7 @@ body[noipspls] .tag_key_ip {
           options.buttons.splice(idx, 1);
         }
       }
-    }
-    catch (ex) {
+    } catch (ex) {
```
Comment (RealDolos) [inline admin.iptools.user.js]: 🙆🏽‍♂️

File: admin.iptools.user.js [hunk 11]
```diff
@@ -244,18 +242,15 @@ body[noipspls] .tag_key_ip {
...
             }
           }
         }
-      }
-      catch (ex) {
+      } catch (ex) {
         console.error(ex);
-      }
-      finally {
+      } finally {
```
Comment (RealDolos) [inline admin.iptools.user.js]: 🤦🏻‍♀️

File: admin.iptools.user.js [hunk 12]
```diff
@@ -244,18 +242,15 @@ body[noipspls] .tag_key_ip {
...
-              else {
+              } else {
                 l.push(el);
               }
             }
           }
         }
-      }
-      catch (ex) {
+      } catch (ex) {
```
Comment (RealDolos) [inline admin.iptools.user.js]: 🤦🏻‍♀️

File: admin.iptools.user.js [hunk 13]
```diff
@@ -372,9 +385,14 @@ body[noipspls] .tag_key_ip {
 
   _templates.forms.freeformselect = {
     title: "Select files from freeform text",
-    fields: [
-      {type: "textarea", name: "freeform", placeholder: " ", label: "Text", rows: "8", cols:"30"},
-    ],
+    fields: [{
```
Comment (RealDolos) [inline admin.iptools.user.js]: 🙆🏽‍♂️

**Review by RealDolos — CHANGES_REQUESTED** [review]
>unrelated formatting changes
>bad ones

b-bls

**Comment by szero** [comment]
omg i run eslint on this without confile

**Comment by szero** [comment]
I also realized, that `git revert` doesn't revert TO the given commit, it reverts the given commits so I should revert only this https://github.com/RealDolos/volascripts/pull/3/commits/d8fba87c9431cd72958177d6da3f5fe9c8487a48 commit. Merge tool hodge-podged it together somehow tho.

File: admin.iptools.user.js [hunk 14]
```diff
@@ -349,19 +349,38 @@ body[noipspls] .tag_key_ip {
      }
   }
 
+  function tickAll() {
+    RoomInstance.extensions.filelist.filelist.forEach(file => {
+      if (!file.dom.fileElement.classList.contains("file_selected")) {
+        file.dom.controlElement.firstChild.click()
```
Comment (RealDolos) [inline admin.iptools.user.js]: >no semicolon

File: admin.iptools.user.js [hunk 15]
```diff
@@ -349,19 +349,38 @@ body[noipspls] .tag_key_ip {
      }
   }
 
+  function tickAll() {
+    RoomInstance.extensions.filelist.filelist.forEach(file => {
+      if (!file.dom.fileElement.classList.contains("file_selected")) {
```
Comment (RealDolos) [inline admin.iptools.user.js]: test for `file.getData("checked")`;

File: admin.iptools.user.js (hunk 14 above)
Comment (RealDolos) [reply admin.iptools.user.js]: `file.setData("checked", true)` should work betterly actually...
No need to check if it's checked at all.

File: admin.iptools.user.js (hunk 15 above)
Comment (RealDolos) [reply admin.iptools.user.js]: Actually see below...

**Review by RealDolos — CHANGES_REQUESTED** [review]

File: admin.iptools.user.js (hunk 15 above)
Comment (szero) [reply admin.iptools.user.js]: Above akshually

**Pull request merged**